Helper/GenerateShopperConversionId.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Adyen\Payment\Helper;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Adyen\Payment\Helper\Util\Uuid;
+use Magento\Checkout\Model\Session as CheckoutSession;
+
+class GenerateShopperConversionId extends AbstractHelper
```

Comment:
For a better naming convention, could you please use a noun instead of a verb in the class name like `ShopperConversionIdGenerator`? Or even `ShopperConversionId` as we can add more methods to this scope eventually.